scenarios/smf-failure.xml

```diff
@@ -7,32 +7,28 @@
     <group name="Smf-Failure">
         <group name="SF-Setup-Fwd-1">
             <step name="SF-Create-Session-1"
-                  exec="mock-smf-cmd create"/>
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c create --count 1 --baseID 1 --ue-pool 17.0.0.0/24 --nb-addr 140.0.100.1"/>
             <step name="SF-Modify-Session-1" requires="^" delay="1"
-                  exec="mock-smf-cmd modify"/>
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c modify --count 1 --baseID 1 --ue-pool 17.0.0.0/24 --nb-addr 140.0.100.1"/>
         </group>
 
         <step name="SF-Check-Flows-1" requires="SF-Setup-Fwd-1" delay="2"
               exec="onos-cli-grep ${OCI} up4:read-flows 'UL flows=1, DL flows=1'"/>
 
         <step name="SF-Kill-Smf" requires="^"
-              exec="mock-smf-cmd kill"/>
+              exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c disassociate"/>
```

Comment:
I believe the `kill` action we were doing before is different than a simple `disassociate`. What we should do here, is emulate a SMF malfunction, so a non-correct disassociation.

Reply:
Done. I had problems with `docker compose up`.